library/src/scripts/flyouts/NewPostMenu.tsx

```diff
@@ -29,7 +34,7 @@ export interface ITransition {
     transform: string;
 }
 
-function ActionItem({ item, style }: { item: IAddPost; style: ITransition }) {
+function ActionItem({ item, style, aid }: { item: IAddPost; style?: ITransition; aid?: string }) {
```

Comment:
Look at other examples of declaring parameter types. You shouldn't be using ":" after the object, it should be on each prop.

Reply:
To clear up a little bit, we have a clear convention defined for how we do prop types, which is generally:

```ts

// We don't always have something else to extend but we sometimes do.
// Less so now with hooks.
interface IProps extends SomeInterface {
   // Optional comment about the prop. Try to add these when you can.
   propName: Type;
}

function MyComponent(props: IProps) {
	const { ... } = props; // Destructuring here.
}


```

Reply:
Got it! Will do again!

Reply:
> To clear up a little bit, we have a clear convention defined for how we do prop types, which is generally:
> 
> ```ts
> // We don't always have something else to extend but we sometimes do.
> // Less so now with hooks.
> interface IProps extends SomeInterface {
>    // Optional comment about the prop. Try to add these when you can.
>    propName: Type;
> }
> 
> function MyComponent(props: IProps) {
> 	const { ... } = props; // Destructuring here.
> }
> ```

Ok, thanks for the info!